public/angular-app/configuration/configuration.html

```diff
@@ -127,22 +128,42 @@ <h3 class="panel-title">Files</h3>
                                                 </li>
                                             </ul>
                                         </div>
+                                        </div>
+                                        <div class="col-md">
+                                            <div class="input-group" style="padding-bottom: 10px; padding-left: 10px;">
+                                                <input id="number" type="text" class="form-control input-sm" value="" ng-keyup="vm.setFilesPage(1, vm.numItems)" ng-model="vm.searchString">
+                                                    <span class="input-group-btn">
+                                                    <button class="btn btn-sm btn-default" type="button" ng-click="vm.setFilesPage(1, vm.numItems)">
+                                                        <i class="fas fa-search"></i>
+                                                    </button>
+                                                </span>
+                                            </div>
+                                        </div>
                                     </div>
 
                                     <!--File selection-->
-                                    <div class="btn-group" style="width: 100%;">
-                                        <button class="btn btn-default dropdown-toggle scrollable-menu-centered" data-toggle="dropdown">
-                                            <span class="caret"></span>
-                                            Files ({{filtered.length}})
-                                        </button>
-                                        <ul class="dropdown-menu scrollable-menu-centered" style="padding-right: 0;">
-                                            <li ng-repeat="file in vm.files | filter: vm.filterFn as filtered" style="padding-right: 5px; padding-left: 5px;">
-                                                <a href="" ng-click="vm.newFile = file.centralPath" style="padding-left: 0; padding-right: 0;">
-                                                    {{file.name | uppercase}}
-                                                </a>
-                                            </li>
-                                        </ul>
-                                    </div>
+                                    <table class="table table-hover" style="width: 100%">
```

Comment:
Let's use DataTables for all tables. I got a ton of examples for all sorts of implementations.